src/ResourceManagement/TrafficManager/TrafficManagerProfileImpl.cs

```diff
@@ -253,6 +268,14 @@ public TrafficManagerProfileImpl WithHttpsMonitoring(int port, string path)
             return this;
         }
 
+        public TrafficManagerProfileImpl WithFastFailover(long? intervalInSeconds, long? timeoutInSeconds, long? toleratedNumberOfFailures = default(long?))
+        {
+            Inner.MonitorConfig.IntervalInSeconds = intervalInSeconds;
```

Comment:
I think parameter does not need `?`, and should check if `MonitorConfig` is null

Reply:
User is able to set null to use default value on service side. The interface does not encourage that (hence only the last param is optional), but I would still let user to set null if they really want that.

I will create Inner.MonitorConfig if null.